idd/Energy+.idd.in

```diff
@@ -104640,35 +104640,45 @@ PythonPlugin:SearchPaths,
        \key Yes
        \key No
        \default Yes
-   A4, \field Search Path 1
+   A4, \field Add epin Environment Variable to Search Path
```

Comment:
New input field to allow responding to the `epin` environment variable, defaulted to ON.  Of course incremented all following input field numbers.